src/components/GameMode/GameMode.scss

```diff
@@ -0,0 +1,11 @@
+.Btn_bg{
```

Comment:
I did some digging about CSS modules naming conventions and I found this guide: https://www.triplet.fi/blog/practical-guide-to-react-and-css-modules/

We should try to stick to these conventions so we are using it correctly (or like everyone else). Classes should be all lowercase and use camelCase. 

So `.Btn_bg` should be `.btnBg`. Not a big enough deal to go and change everything, but lets try to stick to that moving forward.